tests/test_other.py

```diff
@@ -2296,6 +2296,30 @@ def clean(txt):
     out3, err3 = Popen([PYTHON, FILE_PACKAGER, 'test.data', '--preload', 'data2.txt', 'data2.txt@waka.txt'], stdout=PIPE, stderr=PIPE).communicate()
     out3 = clean(out3)
     assert out != out3
+    # verify '--separate-metadata' option produces separate metadata file
+    os.chdir('..')
+    Popen([PYTHON, FILE_PACKAGER, 'test.data', '--preload', 'data1.txt', '--preload', 'subdir/data2.txt', '--js-output=immutable.js', '--separate-metadata']).communicate()
+    assert os.path.isfile('immutable.js.metadata')
+    # verify js output file is immutable when metadata is separated
+    shutil.copy2('immutable.js', 'immutable.js.copy')
```

Comment:
I use copy2() instead of move() or copy() for a good reason. I want to mimic the use case of the packager being invoked multiple times to update the data file (and metadata file) without touching the content of js file and even its timestamp unnecessarily, i.e it is immutable.